src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -91,6 +91,30 @@
     </BoolProperty.Metadata>
   </BoolProperty>
 
+  <!-- TODO create fwlink for this HelpUrl, and find or create a better documentation source -->
+  <EnumProperty Name="DebugType"
+                DisplayName="Debug symbols"
+                Description="Specifies the kind of debug symbols produced during build."
+                HelpUrl="https://github.com/dotnet/roslyn/blob/main/docs/compilers/CSharp/CommandLine.md"
+                Category="General">
+    <EnumProperty.DataSource>
+      <DataSource HasConfigurationCondition="False" />
+    </EnumProperty.DataSource>
+    <EnumValue Name="none"
+               DisplayName="No symbols are emitted" />
+    <!--
+    Note that 'pdbonly' is the same as 'full'.
+    <EnumValue Name="pdbonly"
```

Comment:
This might show up by default for the web SDK, if I'm reading this correctly.

https://github.com/dotnet/sdk/blob/a30e465a2e2ea4e2550f319a2dc088daaafe5649/src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.props#L18

Reply:
Good find. I'll look into converting `pdbonly` into `full` on the way from the project file to the UI.

I also filed https://github.com/dotnet/sdk/pull/16927 to change the web SDK's default `DebugType`.

Reply:
I added interception that allows the UI to read `pdbonly` and display the equivalent value as `full` without having duplicate entries in the drop down. The project file remains unchanged unless the user changes the selection.